x-pack/plugins/saved_objects_tagging/public/management/types.ts

```diff
@@ -18,6 +18,10 @@ export interface TagBulkAction {
    * The label displayed in the bulk action context menu.
    */
   label: string;
+  /**
+   * Optional aria-label if the visual label isn't descriptive enough.
+   */
+  'aria-label'?: string;
```

Comment:
This change is only required if you can manage to [add `count` to bulk actions](https://github.com/elastic/kibana/pull/82816#discussion_r521694329).